glotaran/analysis/optimize.py

```diff
@@ -173,7 +173,7 @@ def _create_result(
         jacobian=jacobian,
         number_of_data_points=number_of_data_points,
         number_of_jacobian_evaluations=number_of_jacobian_evaluation,
-        number_of_variables=number_of_variables,
+        number_of_parameters=number_of_variables,
```

Comment:
Given this change, was it considered changing the line in which number_of_variables is defined? 
`number_of_variables = ls_result.x.size if success else None` # line 122
Might save some confusion down the line?

Reply:
Good point, fixed in [d4efb4f](https://github.com/glotaran/pyglotaran/pull/1012/commits/d4efb4f36b5e82ba6de7b7c75dbc37e4fab6db09)